source/robot_model/install.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+SOURCE_PATH=${PWD}
+
+apt-get update && apt-get install -y \
+    libeigen3-dev \
+    zlib1g-dev \
+    libssl-dev \
+    libboost-all-dev \
+    libboost-dev \
+    libboost-filesystem-dev \
+    libboost-system-dev \
+    libboost-chrono-dev \
+    libboost-date-time-dev \
+    libboost-thread-dev \
+    libboost-test-dev \
+    libbz2-dev \
+    libncurses-dev \
+    pax \
+    tar \
+    liburdfdom-dev \
+    libassimp-dev \
+    assimp-utils \
+    doxygen \
+    texlive-latex-extra
+
+# install googletest
+mkdir ${SOURCE_PATH}/lib
+cd ${SOURCE_PATH}/lib && git clone https://github.com/google/googletest.git
+
+# install pinocchio
+cd ${SOURCE_PATH}/lib && git clone git://git.openrobots.org/robots/robotpkg
+cd ${SOURCE_PATH}/lib/robotpkg/bootstrap && ./bootstrap --prefix=${SOURCE_PATH}/lib/openrobots
```

Comment:
I get a loooot of `deprecated-copy` and `ignored-qualifiers` warnings in this step... can they be suppressed (or preferably, can we check the compiler version and settings against what the pkg expects?

And it does indeed take much, much longer than I realistically anticipated. I believe we will need to break this script into stages (installs dependencies vs install robot_model itself)

Reply:
Precompiled Pinocchio might change that. Worse we can indeed separate that and benefit from caching as you suggested.